OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1306,11 +1306,11 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 
 							if (ibfNode.Any())
 								rh.Value.Nodes.RemoveAll(n => ibfNode.Contains(n));
-								Console.WriteLine("The 'ImagesByFullness' property from the removed RenderHarvester trait has been");
```

Comment:
You fixed the indention error the wrong way here. https://github.com/OpenRA/OpenRA/pull/8644/ did it properly already.